Procfile

```diff
@@ -0,0 +1 @@
+web: gunicorn djangorest.wsgi
```

Comment:
:thinking:  I think this should be `authors.wsgi`

Reply:
I've updated accordingly thanks for pointing out that

Reply:
:+1: Now you can work on indentation and linting issues. Are you using flake8?

Reply:
There's this one issue
![image](https://user-images.githubusercontent.com/12128153/55087893-dbe78380-50bb-11e9-89c9-1d3cf4c445fa.png)
You can install flake8 and autopep8